hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheService.java

```diff
@@ -721,9 +723,14 @@ public CacheEventJournal getEventJournal() {
     // creates the given CacheConfig on all members of the cluster synchronously
     private <K, V> void createCacheConfigOnAllMembers(CacheConfig<K, V> cacheConfig) {
         OperationService operationService = nodeEngine.getOperationService();
-        CacheCreateConfigOperation op = new CacheCreateConfigOperation(cacheConfig);
-        InternalCompletableFuture future =
-                operationService.invokeOnTarget(CacheService.SERVICE_NAME, op, nodeEngine.getThisAddress());
-        future.join();
+        CacheCreateConfigOperation op = new CacheCreateConfigOperation(
+                new PreJoinCacheConfig(cacheConfig, false), true, true);
+        try {
+            InternalCompletableFuture future = operationService.invokeOnTarget(CacheService.SERVICE_NAME, op,
+                    nodeEngine.getThisAddress());
+            future.join();
+        } catch (HazelcastInstanceNotActiveException e) {
+            ignore(e);
```

Comment:
If the operation invocation fails (eg node is `PASSIVE` due to being restored from hot restart), do not fail the cache proxy creation.

Reply:
Maybe it could be worth to have this comment in the code.